packages/react-transform/README.md

```diff
@@ -0,0 +1,119 @@
+# Signals React Transform
```

Comment:
Unsure what's the right level of detail here. Would love other's thoughts.

Reply:
I think it's perfectly fine. We could theoretically shorten the intro paragraph about what signals are by just pointing to the blog post. But then again that I think leaving as is, is fine too.